SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -417,8 +417,28 @@ void FSpatialGDKEditorToolbarModule::StartSpatialOSButtonClicked()
 	const FString ExecuteAbsolutePath = SpatialGDKSettings->GetSpatialOSDirectory();
 	const FString CmdExecutable = TEXT("cmd.exe");
 
+	// Get the correct plugin directory.
+	FString PluginDir = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectPluginsDir(), TEXT("UnrealGDK")));
+
+	if (!FPaths::DirectoryExists(PluginDir))
+	{
+		// If the Project Plugin doesn't exist then use the Engine Plugin.
+		PluginDir = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::EnginePluginsDir(), TEXT("UnrealGDK")));
+	}
+
```

Comment:
Ensure PluginDir exists.

Reply:
We're running UnrealGDK Plugin code at this point. It would be pretty bad if the user had the GDK installed some other way than a project plugin or an engine plugin. I'm pretty sure they wouldn't have been able to hit this point because build wouldn't work....... I'll add regardless.